dynawo/sources/Models/Modelica/Dynawo/Examples/Nordic/Components/GeneratorWithControl/Utilities/InitializedGeneratorSynchronousFourWindings.mo

```diff
@@ -0,0 +1,104 @@
+within Dynawo.Examples.Nordic.Components.GeneratorWithControl.Utilities;
```

Comment:
Same remarks as in ExciterAvrPssOel regarding the comments for parameters, variables, indentation, blank lines, copyright, dynawo types, etc.

Reply:
resolved, see [d146726](https://github.com/dynawo/dynawo/pull/2394/commits/d14672683da0d677556a70ce6fab3936ca672551)